bootstrap/bpdoc/bpdoc_test.go

```diff
@@ -44,3 +44,34 @@ func TestNestedPropertyStructs(t *testing.T) {
 		}
 	}
 }
+
+func TestAllPackages(t *testing.T) {
+	packages, err := AllPackages(pkgFiles, moduleTypeNameFactories, moduleTypeNamePropertyStructs)
+	if err != nil {
+		t.Fatalf("expected no errors, got %s", err)
```

Comment:
Just noticed this: I see that you are using t.Fatalf. t.Fatalf is used mostly for setting up test cases. This looks like a validation step. You should use t.Errorf and let the test case continue since it might discover additional issues. You should be more descriptive about your error:

t.Errorf("got %v, expecting nil error on retrieving all <something> about packages", err)

Reply:
Done.